### ISSUE TITLE
Windows installation

### ISSUE THREAD

**Issue by NeilPearson** [NikolaiT/GoogleScraper#145]
Hi Nikolai,
Just reporting an issue I had installing GoogleScraper on Windows 10/Python3.5. Installation fails unless the line 
package_dir={'examples': 'examples/'},
in setup.py has the forward slash at the end removed. Once that is stripped, everything looks like it works as it should!
Cheers,
Neil

**Comment by dborisog** [comment]
I second Neil. Win7/Python 3.4

**Comment by ddmee** [comment]
Yep, I third that. I'm on Windows 8, Python 3.5. The fix worked. Thanks for posting Neil!

**Comment by rifqiabidin** [comment]
aha gotcha! I'll try your suggestion Neil. Thanks for posting
